src/kokkos/test/kokkos/HistoContainer_t.cc

```diff
@@ -0,0 +1,153 @@
+#include "KokkosCore/kokkosConfigCommon.h"
+#include "KokkosCore/kokkosConfig.h"
+#include <algorithm>
+#include <cassert>
+#include <iostream>
+#include <limits>
+#include <random>
+
+#include "KokkosCore/HistoContainer.h"
+
+template <typename T>
+void go() {
+  std::mt19937 eng;
+  std::uniform_int_distribution<T> rgen(std::numeric_limits<T>::min(), std::numeric_limits<T>::max());
+
+  constexpr int N = 12000;
+  Kokkos::View<T*,KokkosExecSpace> v_d("v_d",N);
+  typename Kokkos::View<T*,KokkosExecSpace>::HostMirror v_h("v_h",N);
```

Comment:
The mirror View can be created also with
```suggestion
  auto v_h = Kokkos::create_mirror_view(v_d);
```